File: PRAK405-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>

int main() {
    int n, k;
    for(int i = 0; i < 3; i++){
    scanf("%d %d", &n, &k);

    int sumAll = 0;

        for (int i = 1; i <= n; i++) {
            int sumLine = 0;
            for (int j = i; j >= 1; j--) {
                int h = j * k;
                sumLine += h;
                printf("(%d * %d)", j, k, h);
                if (j > 1) {
                    printf(" + ");
                }
            }
            sumAll += sumLine;
            printf(" = %d\n", sumLine);
        }

        printf("%d\n", sumAll);
        }
        return 0;
    }

File: PRAK405-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
for i in range(3):
    n, k = map(int, input().split())
    sumAll = 0
    for i in range(1, n+1):
        sumLine = 0
        for j in range(i, 0, -1):
            h = j * k
            sumLine += h
            print(f"({j} * {k})", end="")
            if j > 1:
                print(" + ", end="")
        sumAll += sumLine
        print(f" = {sumLine}")
    print(sumAll)